babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
```

Comment:
- **[OPTIONAL]** Your explanation was just amazing ✨ however, I highly suggest adding some examples or even providing some outer resources so the student can learn something new 👍🏼